### PR TITLE
[WFLY-16619] Convert testsuite/integration/legacy-ejb-client to the j…

### PR DESCRIPTION
**Pull request by bstansberry**
…akarta.* namespace

https://issues.redhat.com/browse/WFLY-16619

See discussion at https://wildfly.zulipchat.com/#narrow/stream/174184-wildfly-developers/topic/testsuite.2Fintegration.2Flegacy-ejb-client.20and.20big-bang

### CODE REVIEW

File: testsuite/integration/legacy-ejb-client/pom.xml [hunk 1]
```diff
@@ -23,73 +23,79 @@
...
             It MUST be excluded to function properly! See the HACK comment in pom.xml</version.org.jboss.ejb-client>
+        <preview.version.org.jboss.ejb-client>The legacy testsuite is unintentionally including ejb client!
+            It MUST be excluded to function properly! See the HACK comment in pom.xml</preview.version.org.jboss.ejb-client>
+
+        <!-- Use the WildFly Preview dependencies until standard WildFly moves to EE 10 -->
+        <dependency.management.import.artifact>wildfly-ee-9-parent</dependency.management.import.artifact>
+        <wildfly.build.output.dir>ee-9/dist/target/${server.output.dir.prefix}-preview-${server.output.dir.version}</wildfly.build.output.dir>
+        <!-- WildFly Arquillian Adapter with Jakarta EE Support -->
+        <version.org.jboss.arquillian.core>1.7.0.Alpha10</version.org.jboss.arquillian.core>
+        <version.org.wildfly.arquillian>5.0.0.Alpha1</version.org.wildfly.arquillian>
```
Comment (chengfang) [inline testsuite/integration/legacy-ejb-client/pom.xml:37]: any reason we're not using 5.0.0.Alpha3, the latest version of wildfly.arquillian?

Comment (bstansberry) [reply testsuite/integration/legacy-ejb-client/pom.xml:37]: Not that I know of. I'm also not sure why Preview is still using a different Arq / WF Arq. I'll ask.

Comment (bstansberry) [reply testsuite/integration/legacy-ejb-client/pom.xml:37]: @chengfang I filed https://issues.redhat.com/browse/WFLY-16651 to track getting everything on the current ARQ, WFARQ and servlet protocol versions. I'd prefer to leave this PR as is though, as the WFLY-16651 task will touch on many poms and I think it's cleaner to just do them all at once.

Comment (chengfang) [reply testsuite/integration/legacy-ejb-client/pom.xml:37]: sounds good.

**Review by chengfang — APPROVED** [review]

**Pull request merged**